app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Authenticate your Kong Gateway Amazon RDS database with AWS IAM
```

Comment:
@windmgc Thanks for drafting this page! I made some content edits and moved a few sections around. If you could look this page over and let me know if I missed anything, that would be great! Thanks!